packages/server/integrations/gitlab/GitLabServerManager.ts

```diff
@@ -38,8 +38,8 @@ class GitLabServerManager {
   ): Promise<[boolean, Error | null]> {
     if (!this.accessToken) return [false, new Error('GitLabServerManager has no access token')]
 
-    const gitLabRequest = this.getGitLabRequest(info, context)
-    const [, error] = await gitLabRequest<GetProfileQuery>(getProfile, {})
+    const gitlabRequest = this.getGitLabRequest(info, context)
```

Comment:
-1\ This is the perfect time to make this conform with the interface that @igorlesnenko built.

Can you say `class GitLabServerManager implements TaskIntegrationManager`?

It'll require you to add a `createTask` method. That method should then internally call `this.getGitLabRequest`.
That way you can just call `manager.createTask()` in createGitLabTask.

you can stub out the other method(s) that the interface requires